src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -132,14 +183,61 @@ class VideoSegmentResource:
     segment_start: float
     segment_end: float
     title: str | None = None
-    comment: list[str] = field(default_factory=list)
-    description: list[str] = field(default_factory=list)
+    comments: list[str] = field(default_factory=list)
+    descriptions: list[str] = field(default_factory=list)
     keywords: list[str] = field(default_factory=list)
     relates_to: list[str] = field(default_factory=list)
     permissions: str = "res-default"
 
     def __post_init__(self) -> None:
         _validate_segment(self)
+        match self.comments:
```

Comment:
I am undecided how I want to treat this. RDU may ignore type hints, therefore it is conceivable that it will not be a list...